src/shogun2-core/shogun2-model/src/main/java/de/terrestris/shogun2/model/UserGroup.java

```diff
@@ -37,6 +37,7 @@
 
 	@ManyToMany
 	@JoinTable(
+		name = "user_usergroup",
```

Comment:
Givinig the JoinTable the same name leads to just one table. So the information is shared and not doubled.